### ISSUE TITLE
Create na_ontap_timezone module 

### ISSUE THREAD

**Issue by joecaruso** [ansible-collections/netapp.ontap#94]
### Summary

It would be nice to have a module that allows you to modify the cluster timezone. 

[REST API](https://library.netapp.com/ecmdocs/ECMLP2882307/html/index.html#/cluster/cluster_modify
):
PATCH /cluster    
"timezone": {"name": "America/New_York"}

CLI:
::> cluster data modify -timezone America/New_York

### Component Name

na_ontap_timezone

### Additional Information

Something that looks similar to this

```yaml (paste below)
- name: Update cluster timezone
  netapp.ontap.na_ontap_timezone:
    hostname: "{{ hostname }}"
    username: "{{ username }}"
    password: "{{ password }}"
    timezone: America/New_York 
```

**Comment by carchi8py** [comment]
Story 5380 is open for this.

**Comment by carchi8py** [comment]
Just a quick note for anyone watching this. 

In rest the Timezone stuff has been moved into the /cluster API. So I'll be adding this to the cluster module as a new field (timezone). So you would be able to change it with a playbook like this when this field get shipped in our next version. 

    - name: modify cluster
      tags: modify
      na_ontap_cluster:
        state: present
        cluster_name: <clustername>
        timezone:
          name: America/Los_Angeles
        <<: *login

**Comment by carchi8py** [comment]
This was fixed in 21.24.0.
